package/CHANGELOG

```diff
@@ -62,6 +62,12 @@ Deprecations
    (PR #4403)
 
 
+03/22/24 leonwehrhan
```

Comment:
move it to the block for release 2.8.0. Add your name to the end of the other authors (of that release)